tests/poly/lower_poly.mlir

```diff
@@ -4,12 +4,25 @@
 #cycl_2048 = #poly.polynomial<1 + x**1024>
 #ring = #poly.ring<cmod=4294967296, ideal=#cycl_2048>
 module {
-  func.func @test_lower_from_coeffs() {
-    %c0 = arith.constant 0 : index
-    // 2 + 2x + 5x^2
-    %coeffs = arith.constant dense<[2, 2, 5]> : tensor<3xi32>
+  func.func @test_lower_add() -> !poly.poly<#ring> {
+    // 2 + 2x + 2x^2 + ... + 2x^{1023}
+    // CHECK: [[X:%.+]] = arith.constant dense<2> : [[T:tensor<1024.*]]
+    %coeffs1 = arith.constant dense<2> : tensor<1024xi64>
+    // CHECK: [[Y:%.+]] = arith.constant dense<3> : [[T]]
+    %coeffs2 = arith.constant dense<3> : tensor<1024xi64>
     // CHECK-NOT: poly.from_coeffs
-    %poly = poly.from_coeffs(%coeffs) : (tensor<3xi32>) -> !poly.poly<#ring>
-    return
+    %poly0 = poly.from_coeffs(%coeffs1) : (tensor<1024xi64>) -> !poly.poly<#ring>
+    %poly1 = poly.from_coeffs(%coeffs2) : (tensor<1024xi64>) -> !poly.poly<#ring>
+    // CHECK: [[XEXT:%.+]] = arith.extui [[X]] : [[T]] to [[TEXT:tensor<1024.*]]
```

Comment:
The width of the element type is computed here and should be asserted. The poly cmod is `2**32` and so while I believe APInt doubles the width to 64, you add one so it may be 65?